[enterprise-4.14] OCPBUGS#33334: Doc improvements related to RHACM procedures (#75582)
openshift/openshift-docs: modules/lvms-deleting-lvmcluster-using-rhacm.adoc

// Module included in the following assemblies:
//
// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc

:_mod-docs-content-type: PROCEDURE
[id="lvms-deleting-lvmcluster-using-rhacm_{context}"]
= Deleting an LVMCluster CR by using {rh-rhacm}

If you have installed {lvms-first} by using {rh-rhacm-first}, you can delete an `LVMCluster` custom resource (CR) by using {rh-rhacm}.

.Prerequisites

* You have access to the {rh-rhacm} cluster as a user with `cluster-admin` permissions.

> * You have deleted the persistent volume claims (PVCs), volume snapshots, and volume clones provisioned by {lvms}. You have also deleted the applications that are using these resources.

Reviewer: Possible format to consider:
```suggestion
* You have deleted the following {lvms}-provisioned resources and any applications that use them:
** Persistent volume claims (PVCs)
** Volume snapshots
** Volume clones
```

Author: Updated as follows:

* You have deleted the following resources provisioned by {lvms}:

** Persistent volume claims (PVCs)
** Volume snapshots
** Volume clones 
+
You have also deleted any applications that are using these resources.

.Procedure

. Log in to the {rh-rhacm} CLI using your {product-title} credentials.

> . Delete the `ConfigurationPolicy` CR YAML file that was created for the `LVMCluster` CR:

Reviewer 2: ```suggestion
. Delete the `ConfigurationPolicy` CR that was created for the `LVMCluster` CR:
```
Here we delete the CR from the cluster not the file.

Author: Updated as per the suggestion.

> . Delete the `ConfigurationPolicy` CR that was created for the `LVMCluster` CR:

Reviewer: ```suggestion
. Delete the `ConfigurationPolicy` CR for the `LVMCluster` CR by running the following command:
```

Author: Updated as per the suggestion.
